pgxn/neon/libpagestore.c

```diff
@@ -623,7 +623,7 @@ pageserver_connect(shardno_t shard_no, int elevel)
 		shard->delay_us = MIN_RECONNECT_INTERVAL_USEC;
 
 		neon_shard_log(shard_no, DEBUG5, "Connection state: Connected");
-		neon_shard_log(shard_no, LOG, "libpagestore: connected to '%s' with protocol version %d", connstr, neon_protocol_version);
+		neon_shard_log(shard_no, DEBUG1, "libpagestore: connected to '%s' with protocol version %d", connstr, neon_protocol_version);
```

Comment:
can we add some log line about `neon_protocol_version` during compute start up?

Reply:
> Isn't that part of the spec, and thus already expected?

just want smth to cross-check version selection logic. Not sure where to add the log statement thought. Can we add it to our extension _PG_init functtion? (iirc setting are already parser by the time _PG_init is called)

Reply:
yes, great, thank you